sdk/storage/azure-storage-file-share/azure/storage/fileshare/_shared/shared_access_signature.py

```diff
@@ -105,12 +110,13 @@ def __init__(self, account_name, account_key, x_ms_version=X_MS_VERSION):
         self.x_ms_version = x_ms_version
 
     def generate_account(self, services, resource_types, permission, expiry, start=None,
-                         ip=None, protocol=None):
+                         ip=None, protocol=None, **kwargs):
```

Comment:
Let's definitely revert all the changes that bring encryption scope to fileshare. Fine to leave constants above but revert everything else to do with encryption scope since its not supported for fileshare and will break things.